src/resources/Microsoft.Authorization/policySetDefinitions/Enforce-Encryption-CMK.json

```diff
@@ -1,364 +1,628 @@
+                }
+            },
+            "MySQLCMKEffect": {
+                "type": "String",
+                "defaultValue": "AuditIfNotExists",
+                "allowedValues": [
+                    "AuditIfNotExists",
+                    "Disabled"
+                ],
+                "metadata": {
+                    "displayName": "Azure MySQL servers bring your own key data protection should be enabled",
+                    "description": "Use customer-managed keys to manage the encryption at rest of your MySQL servers. By default, the data is encrypted at rest with service-managed keys, but customer-managed keys (CMK) are commonly required to meet regulatory compliance standards. CMKs enable the data to be encrypted with an Azure Key Vault key created and owned by you. You have full control and responsibility for the key lifecycle, including rotation and management."
+                }
+            },
+            "PostgreSQLCMKEffect": {
+                "type": "String",
+                "defaultValue": "AuditIfNotExists",
+                "allowedValues": [
+                    "AuditIfNotExists",
+                    "Disabled"
+                ],
+                "metadata": {
+                    "displayName": "Azure PostgreSQL servers bring your own key data protection should be enabled",
+                    "description": "Use customer-managed keys to manage the encryption at rest of your PostgreSQL servers. By default, the data is encrypted at rest with service-managed keys, but customer-managed keys (CMK) are commonly required to meet regulatory compliance standards. CMKs enable the data to be encrypted with an Azure Key Vault key created and owned by you. You have full control and responsibility for the key lifecycle, including rotation and management."
+                }
+            },
+            "SqlServerTDECMKEffect": {
+                "type": "String",
+                "defaultValue": "Deny",
+                "allowedValues": [
+                    "Audit",
+                    "Deny",
+                    "Disabled"
+                ],
+                "metadata": {
+                    "displayName": "SQL servers should use customer-managed keys to encrypt data at rest",
+                    "description": "Implementing Transparent Data Encryption (TDE) with your own key provides increased transparency and control over the TDE Protector, increased security with an HSM-backed external service, and promotion of separation of duties. This recommendation applies to organizations with a related compliance requirement."
+                }
+            },
+            "HealthcareAPIsCMKEffect": {
+                "type": "String",
+                "defaultValue": "audit",
+                "allowedValues": [
+                    "audit",
+                    "disabled"
+                ],
```

Comment:
casing inconsistency

Reply:
Consistent with the policy (which is case-sensitive :))